tests/test_simplicial_complex.cpp

```diff
@@ -233,38 +223,35 @@ TEST_CASE("open_star", "[SC][star]")
     Tuple t(0, 2, -1, 1, hash);
 
 
-    SimplicialComplex sc_v = SimplicialComplex::open_star(Simplex(PrimitiveType::Vertex, t), m);
+    SimplicialComplex sc_v = SimplicialComplex::open_star(Simplex(PV, t), m);
     REQUIRE(sc_v.get_simplices().size() == 8);
 
-    SimplicialComplex sc_e = SimplicialComplex::open_star(Simplex(PrimitiveType::Edge, t), m);
+    SimplicialComplex sc_e = SimplicialComplex::open_star(Simplex(PE, t), m);
     REQUIRE(sc_e.get_simplices().size() == 3);
 
-    SimplicialComplex sc_f = SimplicialComplex::open_star(Simplex(PrimitiveType::Face, t), m);
+    SimplicialComplex sc_f = SimplicialComplex::open_star(Simplex(PF, t), m);
     REQUIRE(sc_f.get_simplices().size() == 1);
 }
 
-TEST_CASE("closed_star", "[SC][star]")
+TEST_CASE("closed_star", "[simplicial_complex][star][2D]")
 {
     tests::DEBUG_TriMesh m;
     m = tests::three_neighbors();
 
     // get the tuple point to V(0), E(01), F(012)
     long hash = 0;
     Tuple t(0, 2, -1, 1, hash);
-    REQUIRE(m.id(t, PrimitiveType::Vertex) == 0);
-    REQUIRE(m.id(m.switch_tuple(t, PrimitiveType::Vertex), PrimitiveType::Vertex) == 1);
-    REQUIRE(
-        m.id(
-            m.switch_tuple(m.switch_tuple(t, PrimitiveType::Edge), PrimitiveType::Vertex),
-            PrimitiveType::Vertex) == 2);
+    REQUIRE(m.id(t, PV) == 0);
+    REQUIRE(m.id(m.switch_tuple(t, PV), PV) == 1);
+    REQUIRE(m.id(m.switch_tuple(m.switch_tuple(t, PE), PV), PV) == 2);
 
 
-    SimplicialComplex sc_v = SimplicialComplex::closed_star(Simplex(PrimitiveType::Vertex, t), m);
+    SimplicialComplex sc_v = SimplicialComplex::closed_star(Simplex(PV, t), m);
```

Comment:
would it make sense to have a 
```
static Simplex Simplex::vertex(const Tuple& t) { return Simplex(PrimitiveType::Vertex, t); }
```
to simplify these incredibly frequent circumstances?

Reply:
Sounds reasonable. I'll also do that after finishing this PR.

Reply:
Whatever, I'll include all the stuff in this PR right away..

Reply:
done